### ISSUE TITLE
Partial Resists Occur When They Shouldn't

### ISSUE THREAD

**Issue by Mango-Ray** [LightsHope/server#1232]
Ever since the latest update that is 1.9, as a warlock, i have had Shadow Bolt give partial resists on mobs who are 61+, have under 75 shadow resistance, and Curse of Shadow applied.

I check resistances of a target in-game with the script command
`/script u=UnitResistance y="target" a=u(y ,0) h=u(y ,1) f=u(y ,2) n=u(y ,3) fr=u(y ,4) s=u(y ,5) z=u(y ,6) print(UnitName(y).." has "..a.." Armor, "..f.." Fire, "..n.." Nature, "..z.." Arcane, "..fr.." Frost and "..s.." Shadow res.", "Emote");
`
When i use this script after Curse of Shadows is applied on such targets, it does indeed show as 0 Shadow resistance, but i still get partial resists.

**Comment by stesve** [comment]
There is a levelbased chance for partial resists, when the mob is higher level than you.

**Comment by Mango-Ray** [comment]
You're indeed correct stesve, for every level your target is above you it gains 5 resistances to every school. Spell penetration from gear and warlock curses negate this.

**Comment by stesve** [comment]
Nope, you cant negate it. This has discussed many times, mostly by paladins and the fact that there is no holy resistance.

https://www.graffes.com/forums/showthread.php?40025-Partial-resists-and-spell-penetration-on-lvl-61-mobs-(caster-glancing-blows)

**Comment by stesve** [comment]
> for every level your target is above you it gains 5 resistances

Its actually 8.

**Comment by theloras** [comment]
_"it gains 5 resistances to every school. However, spell penetration from gear and warlock curses negate the added resistances in addition to whatever base resistances they have."_

Spell Penetration **CANNOT** remove level based resistances

plus what @stesve above

in Vanilla it's 5 resistance for each level - in TBC it was bumped to 8 iirc
